README.md

```diff
@@ -18,6 +16,7 @@
 [Twitter](https://twitter.com/geteslint) |
 [Discord](https://eslint.org/chat) |
 [Mastodon](https://fosstodon.org/@eslint)
+[Bluesky](https://bsky.app/eslint.org)
```

Comment:
```suggestion
[Bluesky](https://bsky.app/profile/eslint.org)
```

`https://bsky.app/eslint.org` doesn't show eslint for me. Perhaps we should use this URL we're using on eslint.org?